docs/src/man/tensors.md

```diff
@@ -947,7 +947,7 @@ the keyword argument `p`. The default value `notrunc()` implies no truncation, a
     dimension of the internal vector space is no larger than `χ`;
 
 *   `truncspace(W)`: truncates such that the dimension of the internal vector space is
-    smaller than that of `W` in any sector, i.e. with
+    smaller than or equal to that of `W` in any sector, i.e. with
```

Comment:
```suggestion
    no bigger than that of `W` in any sector, i.e. with
```
also possible

Reply:
🤷

Reply:
Or 'not greater than', or 'not higher than'. It seems all f I now also start to wonder about the formality of 'no' versus 'not. 

It seems Julia docs describes `<=` and `>=` "Less-than-or-equals" and "greater-than-or-equals". I also find that 'larger' sometimes refers to the absolute value, but since dimensions are positive that is still valid. The mysteries of the English language.

Reply:
I went with 'no greater than'. I think 'greater' is the most appropriate one in the end, but I do think 'no' fits better than 'not' in this kind of phrase.